composer.json

```diff
@@ -70,20 +70,21 @@
         "phpstan/phpstan-phpunit": "^1.1",
         "phpstan/phpstan-webmozart-assert": "^1.2",
         "phpunit/phpunit": "^9.5",
+        "rector/rector": "^0.18.2",
```

Comment:
Should it depend on https://github.com/Sylius/SyliusRector instead? Then explicit `rector/rector` requirement is not needed anymore.

I'm not fully sure why syliusrector exists though.

Reply:
hum sylius rector was exists to add some custom rules to upgrade versions. But I think it's not used anymore.